.travis.yml

```diff
@@ -6,8 +6,13 @@ python:
 install:
     - pip install -r dev_requirements.txt
     - pip install -r requirements.txt
+    - pip install coveralls
 
 script:
     - make lint
     - make docbook
     - make unit
+    - make coverage
```

Comment:
We can probably just run trial with coverage instead of both make unit and make coverage?  And `make coverage` also builds the html, which I think coverage doesn't need and maybe isn't that useful on travis anyway.

Mimic [just runs `coverage run trial` and `coverage report`](https://github.com/rackerlabs/mimic/blob/master/tox.ini#L8) for instance, rather than just run trial for its tests.